src/components/ResultsItem/_results-item.scss

```diff
@@ -19,6 +19,10 @@
 .cc-result-item__description p {
   margin-bottom: 0;
   margin-top: 0;
+
+  .cc-listing {
```

Comment:
is it ok to do this? I used `<Listing>` component inside `<ResultsItem>` 
https://github.com/wellcometrust/corporate-react/pull/566/files#diff-1f5f3ff9f66e63c19855c01c36aba3f0454b1b7c44f044a6d00c8647114cd6dcR143

Reply:
It's overkill really - see https://github.com/wellcometrust/corporate-react/pull/566/files#r519937494